src/test/java/nl/tudelft/ewi/devhub/webtests/views/PullRequestOverViewView.java

```diff
@@ -0,0 +1,88 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static org.junit.Assert.assertTrue;
+
+@Slf4j
+public class PullRequestOverViewView extends PullRequestView {
+
+	public PullRequestOverViewView(WebDriver driver) {
+		super(driver);
+	}
+
+	@Override
+	protected void invariant() {
+		super.invariant();
+		assertTrue(currentPathContains("/pull/"));
+	}
+
+	/**
+	 * Check if the pull request is open.
+	 * @return Returns true if the pull request is open.
+     */
+	public boolean isOpen() {
+		try {
+			WebElement closeButton = getDriver().findElement(By.id("btn-close"));
+			return closeButton.isEnabled();
+		} catch (NoSuchElementException e) {
+			log.warn(e.getMessage(), e);
```

Comment:
Catching exceptions in tests is actually quite bad! Two things can happen:
* The element was expected to be not existing, a false warning is being logged.
* The element was not present while it was expected, but the test still passes
Which behaviour is intended is very hard to read and more importantly, error prone.

What makes a pull request open? When are these buttons visible? From the logic we obtain:

| Pull Request | Is Merged? | Branch Present | Button 1 | Button 2 |
| --- | --- | --- | --- | --- |
| open | - | - | `#btn-close` | `#btn-merge` |
| closed | true | true | `#btn-remove-branch` | `#btn-success:disabled` |
| closed | false | true | `#btn-remove-branch` | `#btn-danger:disabled` |
| closed | true | false | - | `#btn-success:disabled` |
| closed | false | false | - | `#btn-danger:disabled` |

We see that a pull request can only be open if a `#btn-close` is present, and want a safe (non-throwing), descriptive
 way to check this.

Try:
```java
getDriver().findElements(By.id("btn-close"))..stream().findAny().isPresent();
```

Try to clean up the codes for `#isClosed()` and `#isMerged()` as well 😄